File: 2006Scape Server/plugins/plugin/npc/fadli/SecondClick.java
package plugin.npc.fadli;

import com.rs2.event.EventContext;
import com.rs2.event.EventSubscriber;
import com.rs2.event.SubscribesTo;
import com.rs2.event.impl.NpcSecondClickEvent;
import com.rs2.game.players.Player;

import static com.rs2.game.content.StaticNpcList.FADLI;

@SubscribesTo(NpcSecondClickEvent.class)
public final class SecondClick implements EventSubscriber<NpcSecondClickEvent> {

    @Override
    public void subscribe(EventContext context, Player player, NpcSecondClickEvent event) {
        if (event.getNpc() == FADLI) {
            player.getPacketSender().openUpBank();
        }
    }

}

File: 2006Scape Server/plugins/plugin/npc/fadli/SecondClick.kt
package plugin.npc.fadli

import com.rs2.event.EventContext
import com.rs2.event.EventSubscriber
import com.rs2.event.SubscribesTo
import com.rs2.event.impl.NpcSecondClickEvent
import com.rs2.game.players.Player
import com.rs2.game.content.StaticNpcList.FADLI

@SubscribesTo(NpcSecondClickEvent::class)
class SecondClick : EventSubscriber<NpcSecondClickEvent> {

    override fun subscribe(context: EventContext, player: Player, event: NpcSecondClickEvent) {
        if (event.npc == FADLI) {
            player.packetSender.openUpBank()
        }
    }
}